plugins/mac-vth264/encoder.c

```diff
@@ -872,6 +963,17 @@ static obs_properties_t *vt_h264_properties(void *unused)
 	obs_properties_t *props = obs_properties_create();
 	obs_property_t *p;
 
+	p = obs_properties_add_list(props, "rate_control", TEXT_RATE_CONTROL,
+				    OBS_COMBO_TYPE_LIST, OBS_COMBO_FORMAT_INT);
+
+#if defined(__aarch64__)
+	if (__builtin_available(macOS 13.0, *)) {
+		obs_property_list_add_int(p, TEXT_CBR, H264RateControlCBR);
+	}
+#endif
```

Comment:
Does CBR work on Rosetta?
If so, it would be nice if it was also available on x86 builds running on ARM.

Our Rosetta detection is currently part of the UI, if you confirm that CBR would work via Rosetta I'll move it to libobs (which is planned anyways) so that this PR doesn't need to implement it itself.